packages/flutter_driver/test/flutter_driver_test.dart

```diff
@@ -35,8 +35,8 @@ void main() {
       when(mockVM.isolates).thenReturn([mockIsolate]);
       when(mockIsolate.loadRunnable()).thenReturn(mockIsolate);
       when(mockIsolate.invokeExtension(any, any))
-          .thenReturn(new Future.value({'status': 'ok'}));
-      vmServiceConnectFunction = (_) => new Future.value(mockClient);
+          .thenReturn(new Future<Map<String, dynamic>>.value(<String, String>{'status': 'ok'}));
```

Comment:
This extra type ceremony is quite useless in this case. The original code was more readable. I think we should relax our rules for some things.